primary/Cargo.toml

```diff
@@ -28,6 +28,7 @@ tokio-util = { version = "0.7.3", features = ["codec"] }
 tonic = "0.7.2"
 tower = "0.4.12"
 tracing = "0.1.34"
+dhat = "0.3"
```

Comment:
I think it's worth making the dependency optional, and triggered by the feature.
```
[dependencies]
foo = { version = "1.0", optional = true }
bar = { version = "1.0", optional = true }

[features]
fancy-feature = ["foo", "bar"]
```